cfme/networks/network_port.py

```diff
@@ -0,0 +1,90 @@
+from navmazing import NavigateToSibling, NavigateToAttribute
+
+from cfme.common import WidgetasticTaggable
+from cfme.networks.views import NetworkPortDetailsView, NetworkPortView
+from utils import version
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+
+
+class NetworkPortCollection(Navigatable):
+    """ Collection object for NetworkPort object
+        Note: Network providers object are not implemented in mgmt
+    """
+    def __init__(self, appliance=None, parent_provider=None):
```

Comment:
**REQUIRED**
Here and below you've overridden `Navigatable` behaviour, but you should extend it. Please add `Navigatable.__init__(self, appliance=appliance)` and remove `self.appliance = appliance`.